pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
   <PropertyGroup>
-    <Version>1.1.0</Version>
+    <Version>1.1.1</Version>
```

Comment:
Without making the change similar to what @weshaggard made to packages.builds in the referenced diff, we will be generating all packages when we should not be in the servicing branch.

Thus, we should make https://github.com/dotnet/core-setup/blob/release/1.1.0/pkg/projects/packages.builds#L6 to false and add Microsoft.NETCore.App to https://github.com/dotnet/core-setup/blob/release/1.1.0/pkg/projects/packages.builds#L14 since that is the minimal set of packages that will require rebuild (as its project.json will be updated to consume the updated CoreCLR from 1.1.0 branch).